---
title: "컴파일러 오류 CS1017"
ms.date: 07/20/2015
ms.prod: .net
ms.technology:
- devlang-csharp
ms.topic: article
f1_keywords:
- CS1017
helpviewer_keywords:
- CS1017
ms.assetid: e0902e8a-b042-4711-a8a6-83456a3f88cb
caps.latest.revision: 
author: BillWagner
ms.author: wiwagn
ms.openlocfilehash: adfebf5f696090aa7355025e4c2cba2240e1a6c3
ms.sourcegitcommit: 83dd5ec003e788ccb3e33f3412a7af39ae347646
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 03/15/2018
---
# <a name="compiler-error-cs1017"></a>컴파일러 오류 CS1017
Catch 절은 try 문의 일반 catch 절 뒤에 올 수 없습니다.  
  
 매개 변수를 사용하지 않는 `catch` 블록은 일련의 `catch` 블록에서 마지막이어야 합니다. 예외에 대한 자세한 내용은 [Exception Handling Statements](../../csharp/language-reference/keywords/exception-handling-statements.md)을 참조하세요.  
  
## <a name="example"></a>예  
 다음 샘플에서는 CS1017을 생성합니다.  
  
```csharp  
// CS1017.cs  
using System;  
  
namespace x  
{  
    public class b : Exception  
    {  
    }  
  
    public class a  
    {  
        public static void Main()  
        {  
            try  
            {  
            }  
  
            catch   // CS1017, must be last catch  
            {  
            }  
  
            catch(b)  
            {  
                throw;  
            }  
        }  
    }  
}  
```
